### ISSUE TITLE
How can policyValue be 'undefined'?

### ISSUE THREAD

**Issue by smaug----** [w3c/trusted-types#540]
https://w3c.github.io/trusted-types/dist/spec/#abstract-opdef-process-value-with-a-default-policy  step 4
The callbacks are webidl callbacks, so the return value is DOMString? or USVString?, no?

@mbrodesser-Igalia @lukewarlow @koto

**Comment by lukewarlow** [comment]
Yeah I think you're right the undefined would become null through the IDL conversions.

**Comment by petervanderbeken** [comment]
I've commented on this before (https://github.com/w3c/trusted-types/pull/527/files#r1645882026), these algorithms need to start using https://webidl.spec.whatwg.org/#invoke-a-callback-function
